### PR TITLE
Fix build on MSVC, install headers, library, pkgconfig file

### PR DESCRIPTION
**Pull request by nirbheek**
Also build xmllint and install it

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -294,10 +295,23 @@ incdir = include_directories('include')
 xml2lib = library('xml2', sources,
 		  c_args : c_args,
 		  include_directories : incdir,
-		  dependencies: deps)
+		  dependencies: deps,
+                  install: true)
```
Comment (sarum9in) [inline meson.build:299]: Looks like indentation broke here. Probably you should use spaces consistently.

File: meson.build [hunk 2]
```diff
@@ -294,10 +295,23 @@ incdir = include_directories('include')
...
 xml2lib_dep = declare_dependency(link_with: xml2lib,
 				 include_directories: incdir,
 				 dependencies: deps)
```
Comment (sarum9in) [inline meson.build]: Same indentation problem.

File: meson.build [hunk 3]
```diff
@@ -294,10 +295,23 @@ incdir = include_directories('include')
...
+             subdirs: 'libxml2',
+             variables: 'modules=@0@'.format(withcdata.get('WITH_MODULES')))
+
+
+executable('xmllint', 'xmllint.c',
+           dependencies: xml2lib_dep,
+           install: true)
 
 dictexe = executable('testdict', 'testdict.c',
 		     dependencies : xml2lib_dep)
```
Comment (sarum9in) [inline meson.build]: And once again, indentation.

**Review by sarum9in — CHANGES_REQUESTED** [review]
It would also be nice to have `LICENSE.build` but if it is too much hassle to get the consent from other authors feel free to ignore.

**Comment by nirbheek** [comment]
Updated, thanks for noticing the indentation issues!

I'm happy to license my work under the usual `LICENSE.build` used for wraps. CCing other contributors to get their permission too: @TingPing @roshbaby @talisein @jpakkane.

**Comment by TingPing** [comment]
@nirbheek 	You have my permission to re-license my work.

**Comment by talisein** [comment]
You have my permission to re-license my work

**Comment by sarum9in** [comment]
FYI there was a blanket approval from Jussi for this license before for all his work on meson wraps.

**Pull request merged**